src/sentry/tasks/auto_ongoing_issues.py

```diff
@@ -80,3 +80,75 @@ def auto_transition_issues_new_to_ongoing(
             chunk_size=chunk_size,
             expires=datetime.now(tz=pytz.UTC) + timedelta(hours=1),
         )
+
+
+@instrumented_task(
+    name="sentry.tasks.schedule_auto_transition_regressed",
+    queue="auto_transition_issue_states",
+    time_limit=75,
+    soft_time_limit=60,
+)  # type: ignore
+@monitor(monitor_slug="schedule_auto_transition_regressed")
+def schedule_auto_transition_regressed() -> None:
+    now = datetime.now(tz=pytz.UTC)
+    fourteen_days_past = now - timedelta(days=14)
+
+    for project_id in (
+        GroupInbox.objects.filter(
+            date_added__lte=fourteen_days_past,
+            reason=GroupInboxReason.REGRESSION.value,
+        )
+        .distinct()
+        .values_list("project_id", flat=True)
+    ):
+        try:
+            org = Organization.objects.get_from_cache(
+                id=Project.objects.get_from_cache(id=project_id).organization_id
+            )
+        except Exception:
+            org = Organization.objects.get(project_id=project_id)
```

Comment:
Does this work? Organization doesn't have a project_id column.

Reply:
Woops, good catch on this.